internal/core/graphics/image.rs

```diff
@@ -780,4 +788,44 @@ pub(crate) mod ffi {
     ) {
         core::ptr::write(image, Image::from(ImageInner::StaticTextures(textures)));
     }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn slint_image_compare_equal(image1: &Image, image2: &Image) -> bool {
+        return image1.eq(image2);
+    }
+}
+
+/// This structure contains fields to identify and render an OpenGL texture that Slint borrows from the applicatin code.
+/// Use this to embed a native OpenGL texture into a Slint scene, by creating the texture with [`Self::new()`] and
```

Comment:
I wouldn't use Self here in the link as the user would use the fill class name

Reply:
Good idea.